.config-schema.json

```diff
@@ -119,6 +119,33 @@
                 }
             }
         },
+        "metrics": {
+            "type": "object",
+            "properties": {
+                "type": {
```

Comment:
Since we're committing to support `otlp`, but not other exporter formats, I'm not sure it makes sense to have multiple `type` of metrics. We should maybe just change this to



```
"metrics": {
    "type": "object",
    "properties": {
        "enabled" : { "type": "boolean", "default": false },
        "otlp": {...}
    }
}
```

Reply:
Indeed, I've also moved otlp properties to `metrics`, so now it can be configured with following keys:

```yaml
metrics:
   enabled: true
   endpoint: 127.0.0.1:4817
   protocol: http
```